src/ir_Vestel.cpp

```diff
@@ -175,6 +175,43 @@ void IRVestelAC::setAuto(const int8_t autoLevel) {
   else if ( autoLevel == -2 ) setTemp(17);
 }
 
+// Set Timer option of AC.
+// Valid time arguments are 0.5, 1, 2, 3 and 5 hours (in min).
+void IRVestelAC::setTimer(const uint16_t minutes){
+  if( minutes==0 ){
+    remote_state.t_turnOffMinute=0;
+    remote_state.t_timer_mode=1;
+    remote_state.t_on_active=1;
+    remote_state.t_off_active=1;
+    }
+  else{
+    remote_state.t_turnOffHour=minutes/60;
+    remote_state.t_turnOffMinute=minutes/10;
+    remote_state.t_timer_mode=1;
+    remote_state.t_on_active=1; //Yes. t_on_active instead of t_off_active.
+    }
+}
+
+// Set the AC's internal clock
+void IRVestelAC::setTime(const uint16_t minutes){
+  remote_state.t_hour=minutes/60;
+  remote_state.t_minute=minutes-(minutes/60)*minutes;
+}
+
+// Set AC's wake up time. Takes time in minute.
+void IRVestelAC::setWakeupTime(const uint16_t minutes){
+  remote_state.t_turnOnHour=minutes/60;
+  remote_state.t_turnOnMinute=((minutes/60)*minutes-minutes)/10; // Register is 3 bit and can only hold minute/10
```

Comment:
remote_state.t_turnOnMinute= (minutes % 60) / 10;  // Register is 3 bit and can only hold minute/10

Reply:
Yes, we have a "%" right...
In this state of mind. I think I really need to sleep. 7:00am here

Reply:
Get some sleep. This can wait. (and I can make commits without having to merge ;-)